databases/backends/postgres.py

```diff
@@ -150,6 +150,10 @@ def __init__(self, database: PostgresBackend, dialect: Dialect):
         async for row in self._connection.cursor(query, *args):
             yield Record(row, result_columns, self._dialect)
 
+    async def raw_connection(self) -> asyncpg.connection.Connection:
```

Comment:
This doesn't need to (shouldn't) be an async method. (It's not performing any I/O itself, just returning the handle)

I guess we should probably use a property too, rather than a method.

```python
@property
def raw_connection(self):
    ...
```

Reply:
It's possible that the implementations ought to use a `def get_raw_connection()` method, and the developer facing interface should be a `def raw_connection` property, that calls `get_raw_connection()`

Reply:
yeah, was thinking about the property as well, and naming would be more natural as well.